sshkey/sshkey_test.go

```diff
@@ -0,0 +1,48 @@
+package sshkey
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/bitrise-steplib/steps-activate-ssh-key/command"
+	"github.com/bitrise-steplib/steps-activate-ssh-key/log"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestAgentAddKey_CallsSSHAgent(t *testing.T) {
```

Comment:
For me what's confusing about the tests currently is their naming: because it's not straightforward what the test is about. So whenever I need to take a look at it or need to modify it I'll have to reverse engineer the "meaning" of the test from the code - something that is often not straightforward in my experience.

I would suggest we should employ the same BDD-style naming in the test names as we already do in the body here. It would look something like this: `Test_[GivenXyz]_WhenXyz_ThenXyz`. The `Given` part is optional if you are just wiring up the basic dependencies or there's nothing special about those dependencies or their behavior that should be valuable info to the reader.

In this case, I would call this `Test_WhenSSHKeyIsAdded_ThenItCallsSSHAddScript` or something similar.

Wdyt?